dbms/src/Parsers/ExpressionListParsers.cpp

```diff
@@ -231,6 +231,7 @@ bool ParserBetweenExpression::parseImpl(Pos & pos, ASTPtr & node, Expected & exp
     /// For the expression (subject BETWEEN left AND right)
     ///  create an AST the same as for (subject> = left AND subject <= right).
 
+    ParserKeyword s_not{"NOT"};
```

Comment:
it'll look better if all three neighbour variables would be initialized with same brackets, curly or round